lib/compiler-cranelift/src/func_environ.rs

```diff
@@ -573,21 +578,56 @@ impl<'module_environment> BaseFuncEnvironment for FuncEnvironment<'module_enviro
 
     fn translate_table_fill(
         &mut self,
-        _: cranelift_codegen::cursor::FuncCursor<'_>,
-        _: TableIndex,
-        _: ir::Value,
-        _: ir::Value,
-        _: ir::Value,
+        mut _pos: cranelift_codegen::cursor::FuncCursor<'_>,
+        _table_index: TableIndex,
+        _dst: ir::Value,
+        _val: ir::Value,
+        _len: ir::Value,
     ) -> WasmResult<()> {
         Err(WasmError::Unsupported(
             "the `table.fill` instruction is not supported yet".into(),
         ))
     }
 
+    fn translate_ref_null(
+        &mut self,
+        mut pos: cranelift_codegen::cursor::FuncCursor,
+        ty: Type,
+    ) -> WasmResult<ir::Value> {
+        Ok(match ty {
+            Type::FuncRef => pos.ins().iconst(self.pointer_type(), 0),
+            // Type::ExternRef => pos.ins().null(self.reference_type(ty)),
```

Comment:
Optional: generally we don't check in commented out code.

Reply:
It will be useful when we add reference types. I think it will be useful to keep it!